fontforge/autotrace.c

```diff
@@ -25,26 +25,22 @@
  * ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 #include "fontforgevw.h"
```

Comment:
It's in the comments I made earlier, but briefly:
* Get rid of the Windows specific code path by switching to using GLib
* Port the MetaFont importing function to Windows